code/__HELPERS/type2type.dm

```diff
@@ -629,3 +629,9 @@
 			return null
 		r += ascii2text(c)
 	return r
+
+/proc/string2list(str)

```

Comment:
what does this actually do?

Reply:
it turns a string into a list of characters 🤔

Reply:
yes but why, can you not treat strings as lists anyway?

Reply:
wait really?